src/SampleWebApp/Controllers/AuthorssApiController.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SampleWebApp.Data;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/AuthorssApi")]
```

Comment:
Was `AuthorssApi` the the name that was input by the user in this case? You wouldn't normally double-down on "api" in a route.

The `api` prefix is a legacy webapi-ism and isn't required. (Some kind of routing prefix used be be required to use both WebAPI and MVC in the same site). It's not wrong, it's just an old convention based on old requirements. Not sure if others have an opinion.